src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,330 @@
+                    + " | " + task.getBeginDate() + " | " + task.getEndDate() + "\n";
+            appendTaskToDataFile(dataFile, inputToDataFile);
+        } else {
+            System.out.println(EVENT_INVALID_INPUT);
+        }
+    }
+
+    private static void appendTaskToDataFile(FileClass dataFile, String inputToDataFile) {
+        try {
+            dataFile.appendToFile("src/main/data.txt", inputToDataFile);
+        } catch (IOException e) {
+            System.out.println(WRITEFILE_EXCEPTION_MESSAGE);
+        }
+    }
+
+    private static String convertMarkingStatusToNumber(Task task) {
+        if (task.getMarkingStatus().equals("[ ]")) {
+            return "0";
+        } else {
+            return "1";
+        }
+    }
+    private static void changeTaskStatus(ArrayList<Task> tasks, String[] seperatedWords) {
+        try {
+            int lastWordInInteger = Integer.parseInt(seperatedWords[1]);
+            if (lastWordInInteger > tasks.size()) {
+                System.out.println(BIG_NUMBER);
+            } else {
+                boolean doesCommandContainsUnmark = seperatedWords[0].equals("unmark");
+                if (doesCommandContainsUnmark) {
+                    tasks.get(lastWordInInteger - 1).unMark();
+                    System.out.println(FINISH_UNMARKING_MESSAGE);
+                } else {
+                    tasks.get(lastWordInInteger - 1).mark();
+                    System.out.println(FINISH_MARKING_MESSAGE);
+                }
+                System.out.println(lastWordInInteger + ". " + tasks.get(lastWordInInteger - 1).getMarkingStatus()
+                        + " " + tasks.get(lastWordInInteger - 1).getContent());
+            }
+        } catch (Exception error) {
+            System.out.println(INSTRUCTION + "\n" + seperatedWords[0] + ": Number");
+        }
+    }
+    
+    private static void deleteTask(ArrayList<Task> tasks, String[] seperatedWords, FileClass dataFile) {
+        try {
+            int taskNumber = Integer.parseInt(seperatedWords[1]);
+            if (taskNumber > tasks.size()) {
+                System.out.println(BIG_NUMBER);
+            } else {
+                Task item = tasks.get(taskNumber - 1);
+                System.out.println(REMOVE_MESSAGE);
+                if (item.getClassSymbol().equals("[T]")) {
+                    System.out.println(item.getClassSymbol() + item.getMarkingStatus() + " " + item.getContent());
+                } else if (item.getClassSymbol().equals("[D]")) {
+                    System.out.println(item.getClassSymbol() + item.getMarkingStatus() + " " + item.getContent()
+                            + "(by: " + item.getDate() + ")");
+                } else if (item.getClassSymbol().equals("[E]")) {
+                    System.out.println(item.getClassSymbol() + item.getMarkingStatus() + " " + item.getContent()
+                            + "(from: " + item.getBeginDate() + " to: " + item.getEndDate() + ")");
+                }
```

Comment:
Again, instead of checking each type and printing them separately, you can make use of the polymorphism concept I commented above to make your code better structured. Having code like this is not scalable - imagine having to add new type of task into your program, how many methods do you need to change to cater that change currently?